pkg/api/getambassador.io/v2/handwritten.conversion.go

```diff
@@ -658,3 +659,55 @@ func Convert_v3alpha1_TCPMappingSpec_To_v2_TCPMappingSpec(in *v3alpha1.TCPMappin
 
 	return nil
 }
+
+func Convert_v2_TracingServiceSpec_To_v3alpha1_TracingServiceSpec(in *TracingServiceSpec, out *v3alpha1.TracingServiceSpec, s conversion.Scope) error {
```

Comment:
@LukeShu - I looked at the other handwritten conversion functions and it looks like we are using snake case in this file...is there a historical reason? I'm surprised our Linting rules are not yelling about this.